pygmsh/common/geometry.py

```diff
@@ -39,7 +40,13 @@ def __init__(self, env):
         self._OUTWARD_NORMALS = []
 
     def __enter__(self):
-        gmsh.initialize()
+
+        if self.init_argv is None:
+            init_argv = []
+        else:
+            init_argv = self.init_argv
```

Comment:
This can be a one-liner
```
gmsh.initialize([] if self.init_argv is None else self.init_argv)
```

Reply:
Are you able to make that change?
I won't have time for this until next weekend.

Reply:
Okay, made that edit, no trouble.